pyicloud/services/drive.py

```diff
@@ -108,7 +108,7 @@ def _update_contentws(self, folder_id, sf_info, document_id, file_object):
             "command": "add_file",
             "create_short_guid": True,
             "document_id": document_id,
-            "path": {"starting_document_id": folder_id, "path": file_object.name,},
+            "path": {"starting_document_id": folder_id, "path": os.path.basename(file_object.name),},
```

Comment:
Trying to understand why basename() is being called here.
In what situation was the existing code breaking?

Reply:
When you call file_object.name you end up getting the full path, so when pushing to the folder, it failed by not finding the correct destination folder.